ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -88,8 +82,15 @@ public static void scrub(Map<String, String> attributes, ObjectNode json)
       throw new UnwantedDataException(IGNORED_NAMESPACES.get(namespace));
     }
 
-    if (IGNORED_APPS.containsKey(appName)) {
-      throw new UnwantedDataException(IGNORED_APPS.get(appName));
+    if ("FirefoxOS".equals(appName)) {
+      throw new UnwantedDataException("1618684");
+    }
+
+    // event, main, and modules are the only document types that will get aggregated
+    Set<String> docTypes = new HashSet<>(Arrays.asList("event", "main", "modules"));
```

Comment:
This should move to a static variable, named something like `FIREFOX_ONLY_DOCTYPES`